### PR TITLE
Update dailies

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by tobiasKaminsky** [comment]
I just c&p, but this means that NC25 updates to master, but how can I use latest NC25?

File: config/config.php [hunk 1]
```diff
@@ -1075,10 +1075,22 @@
 		],
 	],
 	'daily' => [
-		'23' => [
+		'25' => [
 			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-master.zip',
 			'web' => 'https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html',
 			'eol' => false,
+			'minPHPVersion' => '8.0',
+		],
```
Comment (nickvergessen) [inline config/config.php:1083]:
```suggestion
		'26' => [
			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-master.zip',
			'web' => 'https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html',
			'eol' => false,
			'minPHPVersion' => '8.0',
		],
		'25' => [
			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable25.zip',
			'web' => 'https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html',
			'eol' => false,
			'minPHPVersion' => '7.4',
		],
```

Also while on it, can you fix the eol flags of 23 and lower to true?

**Review by nickvergessen — APPROVED** [review]

**Comment by blizzz** [comment]
please also update integration tests

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**